python/sqlflow_submitter/tensorflow/predict.py

```diff
@@ -39,7 +39,8 @@
     tf.get_logger().setLevel(logging.ERROR)
 else:
     tf.logging.set_verbosity(tf.logging.ERROR)
-    from .pai_distributed import define_tf_flags, make_distributed_info_without_evaluator, dump_into_tf_config
+    from .pai_distributed import define_tf_flags, \
```

Comment:
Don't use backslash line continuation in an `import` statement.

Reply:
Using parentheses for the multi-line import statement.